src/Backend/Buffer.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
```

Comment:
I'd go with a `declare(strict_types=1)`

Reply:
Other services don't use that yet. If it shall be used, it should be used for all services, which would be another PR, and so this is beyond the skope of this PR here.